src/test/kotlin/uk/gov/justice/digital/hmpps/deliusapi/service/contact/CreateContactTest.kt

```diff
@@ -262,18 +270,22 @@ class CreateContactTest : ContactServiceTestBase() {
     havingValidAssociatedEntity: Boolean = true,
     havingValidRarRequirement: Boolean = true,
     havingValidActiveOfficeLocation: Boolean = true,
+    havingValidActiveContactOutcomeType: Boolean = true,
   ) {
     if (!havingValidType) {
       whenever(validationService.validateContactType(request, type))
         .thenThrow(BadRequestException("bad contact type"))
     }
 
     val outcomeMock = whenever(validationService.validateOutcomeType(request, type))
-
-    when (havingValidOutcomeType) {
-      null -> outcomeMock.thenReturn(null)
-      true -> outcomeMock.thenReturn(outcome)
-      false -> outcomeMock.thenThrow(BadRequestException("bad outcome"))
+    if (havingValidActiveContactOutcomeType) {
+      when (havingValidOutcomeType) {
+        null -> outcomeMock.thenReturn(null)
+        true -> outcomeMock.thenReturn(outcome)
+        false -> outcomeMock.thenThrow(BadRequestException("bad outcome"))
+      }
+    } else {
+      outcomeMock.thenCallRealMethod()
     }
```

Comment:
Why do we need to call the real method for an inactive outcome?  Should we not just mock another BadRequestException?

e.g.
```suggestion
    if (havingValidActiveContactOutcomeType) {
      when (havingValidOutcomeType) {
        null -> outcomeMock.thenReturn(null)
        true -> outcomeMock.thenReturn(outcome)
        false -> outcomeMock.thenThrow(BadRequestException("bad outcome"))
      }
    } else {
      outcomeMock.thenThrow(BadRequestException("inactive outcome"))
    }
```

Which could maybe be tidied up a bit, e.g. renaming the var to just `havingActiveOutcomeType`:
```suggestion
    when (havingValidOutcomeType) {
      null -> outcomeMock.thenReturn(null)
      true -> when (havingActiveOutcomeType) {
         true -> outcomeMock.thenReturn(outcome)
         false ->  outcomeMock.thenThrow(BadRequestException("inactive outcome"))
      }
      false -> outcomeMock.thenThrow(BadRequestException("bad outcome"))
    }
```

I guess the same applies to the recent office location changes too?